hack/test-integration.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -euo pipefail
+
+if ! [[ "$0" =~ hack/test-integration.sh ]]; then
+  echo "must be run from repository root"
+  exit 255
```

Comment:
Why do we use 255 as exit code? Here is the list of special meaning of exit code: http://tldp.org/LDP/abs/html/exitcodes.html Maybe we can use 1 as general error

Reply:
Yeah let's use `exit 127` for illegal command, instead.

/cc @kschumy